test/verify/machineslib.py

```diff
@@ -341,7 +341,15 @@ def testBasic(self):
         b.wait_in_text("#vm-subVmTest2-state", "shut off")
 
         # test VM error messages
-        b.wait_visible("#vm-subVmTest2-run")
+        try:
+            with b.wait_timeout(1):
+                b.wait_visible("#vm-subVmTest2-run")
+        except:
```

Comment:
No blanket exceptions. Also this is an extremely tight timeout. Thirdly, test cases  really shouldn't be written in this "opportunistic" and "branched" way. It should be predictable what happens at each step, so they should be linear. 

Either *enforce* that you get an exception (for cases that make sense), or in this case perhaps something like `b.wait_not_present("#vm-subVmTest2-run")` or perhaps `wait_not_visible()` makes more sense?

If there's a good reason for it (I don't see it, I'm not that familiar with what happens here), please add a comment.

Reply:
I was trying to debug the testBasic failure. It should not be in the final code.